website/src/edit-word-panel.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import { HiPencilAlt } from "react-icons/hi"
+import { IoCheckmarkSharp } from "react-icons/io5"
+import {
+  unstable_Form as Form,
+  unstable_FormInput as FormInput,
+  unstable_FormLabel as FormLabel,
+} from "reakit/Form"
+import { unstable_FormSubmitButton as FormSubmitButton } from "reakit/Form"
+import { IconButton } from "./components"
+import { IconTextButton } from "./components/button"
+import * as css from "./edit-word-panel.css"
+import { useForm } from "./form-context"
+import { FormFieldsFragment } from "./graphql/dailp"
+
+/** Button that allows user to enter edit mode in the word panel, and edit fields of a word. */
+export const EditButton = () => {
+  const { form, isEditing, setIsEditing } = useForm()
+
+  return (
+    <Form {...form} className={css.form}>
```

Comment:
I think I want the header of the word details panel to stay the same as it is when you aren't logged in (Selected Word header, syllabary underneath, and close button in top-right corner) while we iterate on that design element. Then when you are logged in, this UI is fine (close button top-left, cancel and save on top-right).